commit 6d0830698922939963f9d7367ecbac0c7f11e176
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Sat Dec 23 22:10:37 2017 +0000

    config: Add PINCTRL_MCP23S08
    
    As of Linux 4.12, the mcp23s08 driver moved from drivers/gpio to
    drivers/pinctrl. At the same time, the Kconfig symbols changed
    from CONFIG_GPIO_MCP23S08 to CONFIG_PINCTRL_MCP23S08, effectively
    removing the MCP23S17 from the downstream defconfigs. Restore
    support by adding CONFIG_PINCTRL_MCP23S08.
    
    See: https://github.com/raspberrypi/linux/issues/2311
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index e3620ef6499e..3cc3a9745c13 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -613,6 +613,7 @@ CONFIG_SPI_SLAVE=y
 CONFIG_PPS=m
 CONFIG_PPS_CLIENT_LDISC=m
 CONFIG_PPS_CLIENT_GPIO=m
+CONFIG_PINCTRL_MCP23S08=m
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_BCM_EXP=y
 CONFIG_GPIO_BCM_VIRT=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index c1da808d40bc..4cfd4e6a3390 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -608,6 +608,7 @@ CONFIG_SPI_SLAVE=y
 CONFIG_PPS=m
 CONFIG_PPS_CLIENT_LDISC=m
 CONFIG_PPS_CLIENT_GPIO=m
+CONFIG_PINCTRL_MCP23S08=m
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_PCF857X=m
 CONFIG_GPIO_ARIZONA=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1157_6d0830698922939963f9d7367ecbac0c7f11e176.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1157_6d0830698922939963f9d7367ecbac0c7f11e176.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
